controllers/argocd/util_test.go

```diff
@@ -389,6 +389,27 @@ func TestGetArgoApplicationControllerCommand(t *testing.T) {
 				"text",
 			},
 		},
+		{
+			"configured operation processors to zero",
+			[]argoCDOpt{operationProcessors(0)},
+			[]string{
+				"argocd-application-controller",
+				"--operation-processors",
+				"10",
+				"--redis",
+				"argocd-redis.argocd.svc.cluster.local:6379",
+				"--repo-server",
+				"argocd-repo-server.argocd.svc.cluster.local:8081",
+				"--status-processors",
+				"20",
+				"--kubectl-parallelism-limit",
+				"10",
+				"--loglevel",
+				"info",
+				"--logformat",
+				"text",
+			},
+		},
```

Comment:
could we also add a test here that checks for `0 < operation processors < 10` ? 
To ensure that the fix addresses the original use case